style/elixir/README.md

```diff
@@ -3,9 +3,91 @@ Elixir
 
 [Sample](sample.ex)
 
+* Don't use a space for padding for a tuple with two or less elements. `{1, 2}`
+* Don't indent pipes when not using the match operator (`=`).
 * Don't separate multiple declarations of functions of the same name with
+* Don't use a space for padding for a list with two or less elements. `[1, 2]`
   newlines.
-* Indent pipes by two spaces when using the match operator (`=`).
-* Don't indent pipes when not using the match operator (`=`).
+* Don't put spaces around matched pairs like brackets, parentheses, etc.
+* Don't use spaces after non-word operators that only take one argument; or 
+  around the range operator.
+* Don't put a space between a function name and the opening parenthesis.
+* Don't use parentheses in macro calls when a do block is passed.
+* Don't prefix macros that return boolean for guards with `is_`
```

Comment:
* Don't put a space between a function name and the opening parenthesis.
_this is a basic programming smell and will be removed_
```elixir
# not preferred
f (3 + 2) + 1
# preferred
f(3 + 2) + 1
```

* Don't use parentheses in macro calls when a do block is passed.
It can be summarized as "Don't include parenthesis in a multi-line function or macro definition."
```elixir
# not preferred
quote(do
  foo
end)
# preferred
quote do
  foo
end
```
* another example not a macro

```elixir
# preferred
Enum.reduce(1..10, 0, fn x, acc ->
  x + acc
end)

# also preferred
Enum.reduce 1..10, 0, fn x, acc ->
  x + acc
end
```